source/includes/_10-one-time-sync-company-custom-check.html.md.erb

```diff
@@ -0,0 +1,587 @@
+        considered to be errored by PassFort, regardless of the other contents
+        of this response.
+      </td>
+    </tr>
+    <tr>
+      <td><code>warnings</code></td>
+      <td>
+        An array containing one or more warnings
+      </td>
+      <td>No</td>
+      <td>
+        Essentially the same as <code>errors</code>, except the
+        <code>sub_type</code> and <code>data</code> fields are not present, and
+        providing warnings will not cause PassFort to consider the check
+        as errored.
+      </td>
+    </tr>
+    <tr>
+      <td><code>provider_data</code></td>
+      <td>
+        Any valid JSON value
+      </td>
+      <td>No</td>
+      <td>
+        This should be the structured JSON data returned by the data provider,
+        or a conversion of the data provider's response, to allow
+        PassFort to investigate issues that may arise with your integration.
+      </td>
+    </tr>
+    <tr>
+      <td><code>charges</code></td>
+      <td>
+        An array containing one or more
+        <a href="#check-charges">Charges</a>
+      </td>
+      <td>No</td>
+      <td>
+        A list of charges incurred while running this check. Charges may only
+        be returned if both of these conditions are met:
+        <ul>
+          <li>No errors occurred (i.e. a result was generated).</li>
+          <li>
+            The check is being resold (the commercial relationship is
+            with PassFort).
+          </li>
+        </ul>
+        
+        For demo checks, charges may be returned, but they will not be deducted
+        from the institution's credit.
+        
+        If charges are returned, the total amount charged must be no greater
+        than the maximum cost-per-check specified in the integration's
+        configuration.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+>This endpoint should return JSON in the following format:
+
+```json
+<%= pretty_json("responses/run_check_sync_company_custom", "json") %>
+```
+#### `check_output`
+
+<table>
+  <thead>
+    <th>Field</th>
+    <th>Type</th>
+    <th>Required?</th>
+    <th>Description</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>decision</code></td>
+      <td>DecisionClass <code></code></td>
+      <td>Yes</td>
+      <td>
+        The check will return one of the following decisions: "PASS | FAIL | PARTIAL | WARN"
+      </td>
+    </tr>
+    <tr>
+      <td><code>summary</code></td>
+      <td>
+      some sort of summary of the result
+        in <a href="#data-structure">the PassFort API</a>.
+      </td>
+      <td>If a match was found by the provider</td>
+      <td>A best effort should be made to populate all fields supported by the provider</td>
+    </tr>
+    <tr>
+        <tr>
+      <td><code>external_resources</code></td>
+      <td>
+        An object 
+        in <a href="#data-structure">the PassFort API</a>.
+      </td>
+      <td>If a match was found by the provider</td>
```

Comment:
We won't require the external resource to be present. It might be something that only makes sense for certain check results (e.g. linking to match details for screening-type checks)